### PR TITLE
Make setup py gets version from file

### LINKED ISSUE #44: version numbers must be collected into one place
Hello, right now package version info and py version info is  not same and all version numbers are needed to change when a bump needed.
So my suggestion is to collect all these info to a seperate file lets name it __version__.py and in that file a variable for example version  , and in whole other places ( in setup.py , in main app py   , ... ) Ve use that variable that will let us fix all version related problems

### PR DESCRIPTION
**Pull request by usta**
I believe with this way it will be much suitable for version bumps.
This PR also fixes a small bug about a typo in MANIFEST.in
fixes #44

### CODE REVIEW

**Comment by timabbott** [comment]
@eeshangarg I assume you'll review (are you subscribed to notifications for this repo such that I don't need to mention you?)

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,2 @@
-include README
+include README.md
 include LICENSE
```
Comment (eeshangarg) [inline MANIFEST.in]: Could you please split this one out into a separate commit?

**Comment by eeshangarg** [comment]
@timabbott: yes, I just subscribed to alerts for this repo, so you don't have to explicitly mention me in issues/PRs anymore! Thanks! :)

File: MANIFEST.in (hunk 1 above)
Comment (usta) [reply MANIFEST.in]: > 
> 
> Could you please split this one out into a separate commit?

Sure

**Comment by usta** [comment]
> 
> 
> @usta: Thanks for working on this! I tested this and it works.
> 
> Now that I think about it, the `__version__` value is kinda useless. The only purpose it serves is printing an error message for when the script isn't given enough command-line arguments. For that, we don't really need the version number. And since the package is never really imported (it only has an executable), I can't imagine a scenario where a user may go `import clonevirtualenv; clonevirtualenv.__version__`, they can just use `pip freeze` if they want to find out the version. Is there a reason why a `__version__` value might be useful for an executable-only package that I am not aware of? If not, we could honestly just get rid of it and that would make everything simpler.
> 
> Even some popular Python packages have a history of using "hacks" to store version numbers globally in an external file, so it isn't a big deal either way, but still worth a short discussion. :) Thanks!

If you ask my opinion it will be much better we just have the version number in setup.py, not in other files.
If you accept I will prepare a striped version of version :)

File: MANIFEST.in (hunk 1 above)
Comment (usta) [reply MANIFEST.in]: @eeshangarg #50 is ready

**Comment by usta** [comment]
@eeshangarg @timabbott also I have stripped down the __version__ info from script file so we just follow version in setup.py   with : #51

**Comment by eeshangarg** [comment]
@usta: Cool sounds good, I just reviewed your other PRs! :) Also, could you please close this PR now that we've decided to go the other way? :)

**Comment by usta** [comment]
We decided to make this PR canceled for benefits of other PRs
